corehq/sql_db/routers.py

```diff
@@ -65,7 +66,9 @@ def allow_migrate(db, app_label):
 
     if app_label == PROXY_APP:
         return db == partition_config.get_proxy_db()
-    elif app_label in (FORM_PROCESSOR_APP, SCHEDULING_PARTITIONED_APP):
+    elif app_label == BLOB_DB_APP and db == 'default':
+        return True
```

Comment:
does this mean that the blob meta tables will be in the main DB too (though unused)?

Reply:
Unfortunately yes, although they will not be used. I didn't see a way around that, since there is no way to mark individual database entities within an app to be sharded since routing is done by app label.